worker/src/queues/ses/types.ts

```diff
@@ -10,4 +13,24 @@ export type SESJob = {
   body: string;
   attachments: FormField[];
   reference: string;
+  postAlertOptions: {
```

Comment:
I'm guessing the thinking is that we should have a staff email service that should do both of the emails to staff, but it feels weird having things named as SES when they're notify commands right?